internal/mode/static/nginx/runtime/manager.go

```diff
@@ -94,6 +117,36 @@ func (m *ManagerImpl) Reload(ctx context.Context, configVersion int) error {
 	return nil
 }
 
+// UpdateHTTPServers uses the NGINX Plus API to update HTTP upstream servers.
+// Only usable if running NGINX Plus.
+func (m *ManagerImpl) UpdateHTTPServers(upstream string, servers []ngxclient.UpstreamServer) error {
+	if !m.IsPlus() {
```

Comment:
I wonder if it is a case of programmer error, for which it is suitable panic, rather then returning an error? similar in  GetUpstreams()